testing/scenario_app/ios/Scenarios/ScenariosUITests/PlatformViewGestureRecognizerTests.m

```diff
@@ -110,9 +110,54 @@ - (void)testAccept {
       [[XCTNSPredicateExpectation alloc] initWithPredicate:predicate object:platformView];
 
   [platformView tap];
+
+  [self waitForExpectations:@[ expection ] timeout:kSecondsToWaitForPlatformView];
+  XCTAssertEqualObjects(platformView.label,
+                        @"-gestureTouchesBegan-gestureTouchesEnded-platformViewTapped");
+}
+
+- (void)testGestureWithMaskViewBlockingPlatformView {
```

Comment:
does this test fails if you comment out FlutterPlatformViews_Internal.mm #L99-102?